### DISCUSSION TITLE
Issue 2595

### DISCUSSION THREAD

**Discussion by Thorn177** [Courseplay/Courseplay_FS22#3324] [Q&A]
I am getting this same situation about 2/3 of the times I go to do any vinework.  It misreads the shape of the field as being from the northwest most pole to the southeast most pole.  I have experimented with vineyards on both a mod map and a vanilla map (elmcreek) with no other mods.  No olive or cloned mods of olive have worked, but I've only done three of them because of the lack of snappage.  Adjusting the row start and finish in the placeables xml file to be along a cardinal point and the correct distance apart for olives doesn't change the outcome.  Being in a field or not doesn't affect it.  Adding or removing rows doesn't change the outcome.  Being on a completely random direction didn't matter either.  I did have one vineyard change when I removed all bar one row and then rebuilt it the original size, change from not reading correctly, to reading correctly.  No matter what I have tried, only 1/3 of the vineyards are read correctly.

**Comment by Tensuko** [comment]
Read this:
https://github.com/Courseplay/Courseplay_FS22/issues/1683
A little search and you would have found it.
